pkg/azurefile/controllerserver.go

```diff
@@ -353,6 +353,17 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	}
 
 	fileShareSize := int(requestGiB)
+
+	if account != "" && sku == "" && fileShareSize < minimumPremiumShareSize {
+		accountProperties, err := d.cloud.StorageAccountClient.GetProperties(ctx, subsID, resourceGroup, account)
+		if err != nil {
+			return nil, status.Errorf(codes.Internal, "failed to get properties on storage account account(%s) rg(%s), error: %v", account, resourceGroup, err)
```

Comment:
pls set this a warning message since ` d.cloud.StorageAccountClient` may have no permission to get properties, and this is a nice to have fix.

Reply:
Ok, changed.